.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:6.10.3
```

Comment:
Not LTS?

Reply:
Take your pick:
https://circleci.com/docs/2.0/circleci-images/#nodejs

Reply:
`- image: circleci/node:latest`?

Reply:
Yeah, I think I like `latest` as well. Kind of surprised they don't have an `LTS` option to track what the current lts is.